app/presenters/summary/sections/income_details.rb

```diff
@@ -8,11 +8,20 @@ def show?
       end
 
       def answers # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
-        [
-          Components::ValueAnswer.new(
-            :income_above_threshold, income.income_above_threshold,
-            change_path: edit_steps_income_income_before_tax_path
-          ),
+        answers = []
+        answers << if include_partner_in_means_assessment?
+                     Components::ValueAnswer.new(
+                       :joint_income_above_threshold, income.income_above_threshold,
+                       change_path: edit_steps_income_income_before_tax_path
+                     )
+                   else
+                     Components::ValueAnswer.new(
+                       :income_above_threshold, income.income_above_threshold,
+                       change_path: edit_steps_income_income_before_tax_path
+                     )
+                   end
+
+        answers << [
```

Comment:
We've handled this in the past by passing a param option to the locale. For example, the way we present the maat id or usn number for appeal no changes cases: https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/main/app/presenters/summary/sections/case_details.rb#L111

Might be cleaner to handle this way